mappings/sweet-envo_cryosphere.sssom.tsv

```diff
@@ -0,0 +1,201 @@
+ENVO:03400003	snow cover	sopropst:SnowCover	snow cover	skos:relatedMatch	semapv:ManualMappingCuration	SWEET has two terms for snow cover (a thickness and a hydrospheric phenomenon);
+ENVO:03000141	snow layer	sophhy:SnowCover	snow cover	skos:relatedMatch	semapv:ManualMappingCuration	SWEET has two terms for snow cover (a thickness and a hydrospheric phenomenon);
+ENVO:03000141	snow layer	sopropst:SnowCover	snow cover	skos:relatedMatch	semapv:ManualMappingCuration	SWEET has two terms for snow cover (a thickness and a hydrospheric phenomenon);
+ENVO:03000142	snow water equivalent	sopropst:SnowWaterEquivalent	snow water equivalent	skos:closeMatch	semapv:ManualMappingCuration	Snow water equivalent is a quality in ENVO and a property in SWEET
+ENVO:01000895	snowfall	sophatmopc:Snowfall	snowfall	skos:closeMatch	semapv:ManualMappingCuration	In ENVO this is a commitment to a water ice precipitation process; while in SWEET it is a type of atmospheric precipitation which is a phenomenon (which is ambiguous).  At the top level these may not be quite the same.
+ENVO:03000116	snowpack	sopropst:Snowpack	snowpack	skos:relatedMatch	semapv:ManualMappingCuration	In SWEET this is a thickness, in ENVO a mass
+ENVO:01000881	snowstorm	sophatmopc:Storm	storm	skos:broadMatch	semapv:ManualMappingCuration	SWEET does not have snowstorm
+ENVO:01001880	subglacial	sophcr:Subglacial	subglacial	skos:relatedMatch	semapv:ManualMappingCuration	SWEET has this in the phenomenon branch while it is a quality in ENVO
+ENVO:01001881	supraglacial	sophcr:Supraglacial	supraglacial	skos:relatedMatch	semapv:ManualMappingCuration	SWEET has this in the phenomenon branch while it is a quality in ENVO
+ENVO:01001534	tabular iceberg	soreac:TabularIceberg	tabular iceberg	skos:closeMatch	semapv:ManualMappingCuration	A candidate for exact match if SWEET adds ENVO's definition to SWEET's term
+ENVO:00000552	talik	soreas:Talik	talik	skos:relatedMatch	semapv:ManualMappingCuration	In SWEET talik is a soil layer; but there is no definition.  In ENVO talik is an area of unfrozen ground surrounded by permafrost.
+ENVO:01001509	terrestrial ice calf	soreac:Calf	calf	skos:broadMatch	semapv:ManualMappingCuration	SWEET Calf is less specific than ENVO terrestrial ice calf
+ENVO:01001611	thawing process	soprocsc:Thawing	thawing	skos:closeMatch	semapv:ManualMappingCuration	Candidate Exact match if SWEET adopts the ENVO definition.
+ENVO:03000085	thermokarst	sorealg:Thermokarst	thermokarst	skos:closeMatch	semapv:ManualMappingCuration	In SWEET this is a landform, which is accurate - need to compare upper level layers to determine if an exact match could be made if SWEET adopted the ENVO definition
+ENVO:00000254	till plain	sorealg:TillPlain	till plain	skos:relatedMatch	semapv:ManualMappingCuration	Despite having identical definitions, the SWEET till plain is a geometrical object; while the ENVO till plain is a material entity

```

Comment:
Hmm, I think this should be close? This is the sweet concept:

<img width="592" alt="image" src="https://github.com/EnvironmentOntology/envo/assets/50745/85b37042-4fd9-41f4-9831-5ee33431109e">

I have seen many cases where sweet confuses things under geometrical branches but this doesn't seem to be one of them?

Reply:
@rduerr where are you seeing that http://cor.esipfed.org/ont?iri=http://sweetontology.net/realmLandform/LandRegion is a subclass of GeometricalObject? The OAK diagram above shows the complete paths to root, and this is consistent with the URIs.